### ISSUE TITLE
Client simularion Android 8.1

### ISSUE THREAD

**Issue by dilyanpalauzov** [ssllabs/ssllabs-scan#749]
See https://github.com/drwetter/testssl.sh/issues/1333: Android 8.1 does not support apparently TLS 1.3, Chrome on Android 8.1 does support TLS 1.3.  The client simulation shall say for Android 8.1 only TLS 1.2.

**Comment by ArchangeGabriel** [comment]
Yeah, every `Android x.y` tests are actually for Chrome. I’ve had multiple issues with that in the past (like a given version of Android not actually supporting some cipher or elliptic curve).

**Comment by schildbach** [comment]
I've run into this bug, too. The "Android" simulations should really be renamed to avoid further confusions.
